app/Filament/App/Resources/LocalIndicatorResource.php

```diff
@@ -54,10 +61,9 @@ public static function table(Table $table): Table
                     ->label('Module')
                     ->sortable()
                     ->searchable(),
-                Tables\Columns\TextColumn::make('globalindicators_count')
-                    ->label('# Global indicators')
-                    ->counts('globalindicators')
-                    ->sortable(),
+                Tables\Columns\TextColumn::make('globalIndicator.name')
+                    ->sortable()
+                    ->searchable(),
```

Comment:
Can this be a select column, so teams can add the global indicators here while seeing their full list of local indicators?